design/multiple-storage-types-support.md

```diff
@@ -106,7 +106,7 @@ Basically, a storage backend defines their custom types and the parent/child rel
 The metacontroller handles all the K8s API interactions and regularly calls into storage backend defined “hooks”.
 The storage backend is given JSON representing the current state in K8s types and then returns JSON defining in K8s types what the desired state should be.
 The metacontroller then makes that desired state a reality via the K8s API.
-This pattern does allow for fairly complicated stateful apps (e.g. [Vitess](https://github.com/GoogleCloudPlatform/metacontroller/tree/master/examples/vitess)) that have well defined parent/children hierarchies, but doesn’t allow the storage backend controller to perform “imperative” operations to manipulate cluster state.
+This pattern does allow for fairly complicated stateful apps (e.g. [Vitess](https://github.com/GoogleCloudPlatform/metacontroller/tree/master/examples/vitess)) that have well defined parent/children hierarchies, and can allow for the storage backend operator to perform “imperative” operations to manipulate cluster state by launching Jobs.
```

Comment:
Are we saying that it is required for ceph to launch jobs to orchestrate its cluster? I thought we decided it was too complex for ceph?